src/dateTimeLocal.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import DateTimeLocalInput from './DateTimeLocalInput';
+
+
+describe('DateTimeLocal', function() {
+
+	let dateTimeComponent,
+		dateTimeInputEl;
+
+	const value = '1999-12-31T23:59:59',
+		newValue = '3000-12-31T23:59:59',
+		min = '1999-12-30T00:00:00',
+		max = '3001-12-31T23:59:59',
+		callbackSpy = jasmine.createSpy('foo');
+
+	beforeEach(() => {
+		dateTimeComponent = TestUtils.renderIntoDocument(
+			<DateTimeLocalInput
+				name='datetime'
+				value={value}
+				callback={callbackSpy}
+				min={min}
+				max={max} />);
```

Comment:
good to align the closing `/>` with opening `<`, and closing `)` with the start of the line that has the opening `(`